aws/modules/eks/outputs.tf

```
@@ -0,0 +1,32 @@
+# Copyright 2021 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Output attributes of the eks cluster.
+# Name of the EKS cluster
+output "cluster_name" {
+  description = "Name of the EKS cluster"
+  value       = aws_eks_cluster.eks_cluster.id
+}
+
+# Version of the EKS cluster
+output "cluster_version" {
+  description = "Version of the EKS cluster"
+  value       = aws_eks_cluster.eks_cluster.version
+}
+
+# The endpoint for your EKS Kubernetes API
+output "cluster_Endpoint" {
+  description = "The endpoint for your EKS Kubernetes API"
+  value       = aws_eks_cluster.eks_cluster.endpoint
+}
```

Missing newline at end of file.

```suggestion
}

```